src/widget/wmainmenubar.cpp

```diff
@@ -362,6 +362,18 @@ void WMainMenuBar::initialize() {
     pOptionsMenu->addAction(pOptionsBroadcasting);
 #endif
 
+    // TrainingMode
+    m_pConfigTrainingmodeEnabled = new ControlObject(ConfigKey("[TrainingMode]", "enabled"));
+    QString TrainingmodeTitle = tr("Enable Training Mode");
+    QString TrainingmodeText = tr("In training mode the play counter will not be increased");
+    auto pOptionsTrainingmode = new QAction(TrainingmodeTitle, this);
```

Comment:
Here you can make use of 
make_parented if you like.

This has just the documentation purpose that the lifetime of this object is managed by the QT object tree.
A explicit delete() would or possible but redundant. It introduces some overhead in object tree management on top.

Reply:
Okay, make sense. Will change that also in the rest of this file. Because it's not done with any of the QT objects there, so we have that also consistent in the file.